website/client/src/components/groups/group.vue

```diff
@@ -509,11 +515,7 @@ export default {
         // Load invites
       }
       await this.fetchGuild();
-      // Fetch group members on load
-      this.members = await this.loadMembers({
-        groupId: this.group._id,
-        includeAllPublicFields: true,
-      });
+
       this.$root.$on('updatedGroup', group => {
```

Comment:
(unrelated but this probably leads to a memory leak as the event listener is never cleared)